app/modules/encounters/resources.py

```diff
@@ -56,82 +53,6 @@ def get(self, args):
         """
         return Encounter.query.offset(args['offset']).limit(args['limit'])
 
-    # NOTE: question whether POST /encounter should be allowed at all, since it would be detached from a sighting!
```

Comment:
Excellent, You have implemented https://wildme.atlassian.net/browse/DEX-286
It's now assigned to you so you can take the credit